src/layouts/partial-cards.html

```diff
@@ -27,7 +27,7 @@
       }
 
       app-header-layout {
-        background-color: #E5E5E5;
+        background-color: var(--paper-grey-200);
```

Comment:
Create a separate variable.
If you want a green background here you wouldn't want to override `paper-grey-200` to green,